workflow/controller/controller.go

```diff
@@ -195,32 +192,6 @@ func (wfc *WorkflowController) Run(ctx context.Context, wfWorkers, podWorkers in
 	<-ctx.Done()
 }
 
-// podLabeler will label all pods on the controllers completedPod channel as completed
-func (wfc *WorkflowController) podLabeler(stopCh <-chan struct{}) {
```

Comment:
I don't *think* we need this, not sure about using channels for this - due to robustness

Reply:
re-implemented elsewhere